--- allauth/account/views.py	2013-10-16 16:29:14.000000000 -0500
+++ allauth/account/views.py	2013-10-16 16:29:54.000000000 -0500
@@ -455,6 +455,7 @@
     def dispatch(self, request, uidb36, key, **kwargs):
         self.uidb36 = uidb36
         self.key = key
+        self.request = request
         self.request.user = self._get_user(uidb36)
         if not self.token_generator.check_token(self.request.user, key):
             return self._response_bad_token(request, uidb36, key, **kwargs)
